software-engineering-policies/TechnicalDebt/TechnicalDebtPolicy.md

```diff
@@ -1,48 +1,172 @@
 
-See the [Technical Debt Guidance](TechnicalDebtGuidance.md) for details on recording this.
+## Technial Debt Impact
 
-## Verification
+Technical debt impacts various aspects of an application's life cycle, this is often dictated by the type of technical debt but common impacts you can expect from outstanding technical debt are:
 
-The team's register of technical debt **MUST** be available for inspection. Each project should have an accessible and clear list of technical debt which should be available in the relevant area of either Azure DevOps or Azure DevOps Server.
+* Increased time to deliver
+* Unplanned work
+* Inaccurate planning
+* Disengaged Development teams
+* Longer times to recover
+* Instability
 
-See the [Technical Debt Guidance](TechnicalDebtGuidance.md) for details on building this capability.
+---
 
-## Additional Reading
+## Raising Technical Debt
 
-​<https://martinfowler.com/bliki/TechnicalDebt.html>
-<https://martinfowler.com/bliki/EstimatedInterest.html>
-<https://martinfowler.com/bliki/TechnicalDebtQuadrant.html>
+Technical debt identified in the current sprint but not being resolved as part of present work **MUST** be raised as a PBI in the relevant area of Azure DevOps. The relevant location is based on whether the technical debt is related to a product or a team.
+
+Ideally even if technical debt is going to be resolved immediately raising it is good for understanding its impact on UKHO work and identifying that technical debt was discovered.
+
+When raising technical debt, please add the following information to the Technical Debt PBI:
+
+### Title
+
+```Technical Debt - {Concise outline of Technical Debt item}```
+
+### Description
+
+The description of the Technical Debt item must include the following as a minimum:
+
+* The repository the Technical Debt lives within (a link to the repo would be good but the name would suffice)
+* Description of the issue in as much detail as you can provide, including class/file names if this can be included
+* A proposed solution or ideas for a fix **
+* Cost/Benefit and risk of the Technical Debt item, simply as a "why should this be fixed?/How long would it take?"
+
+### Tagging Strategy
+
+* The Technical Debt PBI **MUST** be tagged with the `````Technical Debt````` tag and should have any further appropriate tags, such as the following:
+  * Product Name (could be multiple, see the product list in AppRegister; [AppRegister](http://appregister/))
+  * Tech Debt Type:
+    * Security
+    * Knowledge
+    * Automation
+    * Testing
+    * Architecture
+    * Code
+    * Infrastructure
+  * Tech Debt Impact:
+    * Availability
+    * Portability
+    * Compatibility
+    * Performance
+    * Maintainability
+    * Extensibility  
+  * Out-Of-Support**
+  * Investigate To Remove** - This tag has been used to mark a Technical Debt item that may already have been completed or is going to be irrelevant due to other work.
+  * UKHO TD priority tag:
```

Comment:
One issue we may face is object persistance, if something classified as a TD1 you have to fix it ASAP, if you arent looking at it right now, does it move to a TD2? Maybe it is a 1, but as soon as you start the project up this has to be fixed in the first sprint.

Reply:
So the TD tags aren't strict in that sense, it is a tag for the technically focused SMEs to use to inform others when refining or reviewing the piece of TD. This shouldn't change, as the original identifier has flagged it as TD1 so should be being looked at for resolution at every opportunity. 

It has been raised that people might just put everything as TD1 then, my counter point being, are we not meant to all be professional in our work practice?